bin/windows/installChocolatey.cmd

```diff
@@ -0,0 +1,8 @@
+@echo off
```

Comment:
this is an easy install command the Chocolatey pacakge manager for Windows users who already have the repo cloned. Read the README for more info.